packages/appstore/src/components/options-multipliers-listing/index.tsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { observer } from 'mobx-react-lite';
 import { Text, StaticUrl } from '@deriv/components';
+import { ContentFlag } from '@deriv/shared';
+import { useStore } from '@deriv/stores';
 import { Localize, localize } from '@deriv/translations';
 import ListingContainer from 'Components/containers/listing-container';
-import { BrandConfig } from 'Constants/platform-config';
-import TradingAppCard from 'Components/containers/trading-app-card';
-import { useStores } from 'Stores/index';
-import { isMobile, ContentFlag } from '@deriv/shared';
 import PlatformLoader from 'Components/pre-loader/platform-loader';
+import TradingAppCard from 'Components/containers/trading-app-card';
+import { BrandConfig } from 'Constants/platform-config';
 import { getHasDivider } from 'Constants/utils';
 
 const OptionsAndMultipliersListing = () => {
```

Comment:
```suggestion
const OptionsAndMultipliersListing = observer(() => {
```

Reply:
Updated.
observer was already wrapped over the component with the export default line. Reformatted the code to be consistent.